10.1/root/usr/share/container-scripts/mysql/common.sh

```diff
@@ -92,6 +94,7 @@ function initialize_database() {
   # Using empty --basedir to work-around https://bugzilla.redhat.com/show_bug.cgi?id=1406391
   mysql_install_db --rpm --datadir=$MYSQL_DATADIR --basedir=''
   start_local_mysql "$@"
+  mysql_upgrade --socket=/tmp/mysql.sock
```

Comment:
As I'm not familiar with mysql/mariadb "upgrading" procedures, I'm curious why this method changed...  Can we have small comment above this command?